internal/client/consensus/grandpa/environment_test.go

```diff
@@ -615,3 +618,19 @@ func TestVoteEncoding(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, voteVal, newVote)
 }
+
```

Comment:
Are tests being left for future work?

Reply:
Yes, I need the `voterWork` issue #3606 side of things to actually test this since that drives the `grandpa.Voter` in `pkg/finality-grandpa` which calls methods on `environment`.